package-res/ccc/core/base/chart/chart.axes.js

```diff
@@ -433,6 +456,30 @@ pvc.BaseChart
             }
         }
 
+        // CDF603
+        var width;
+        if(axis.option.isDefined('FixedLength')) {
+            width = axis.option('FixedLength');
+            if(width < 0) width = -width; // Length is always an absolute value
+        }
+
+        // CDF603
+        // If FixedMin + FixedLength specified, max is directly set
+        // using both and FixedMax is ignored
+        // if width is not null then FixedLength was defined
+        if(min   != null && axis.option.isDefined('FixedMin') && 
+           width != null && max == null) {
+            max = min - (0-width);
+            maxLocked = (max != null);
+            if(maxLocked) {
+                // this shouldn't be necessary since width is always 
+                // greater than 0 and if scaleUsesAbs returns true,
+                // so is min
+                if(max < 0 && axis.scaleUsesAbs()) max = -max;
```

Comment:
Should we replace this with an assertion then?
I'm also worried that simply switching signs like this can make max < min. Are we counting on the general scale code that detects that case and switches them?